tox.ini

```diff
@@ -74,9 +72,7 @@ deps =
     {[deps-base]deps}
 passenv = *
 extras = all
-setenv =
```

Comment:
please leave this, see above

Reply:
Reverted change in 25abec60251a55b9d5e1a763c5b9be6290e8eef8.